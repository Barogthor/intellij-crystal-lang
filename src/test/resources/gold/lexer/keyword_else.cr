b = 1
if b > 2
  b = 10
else
  b = 20
end