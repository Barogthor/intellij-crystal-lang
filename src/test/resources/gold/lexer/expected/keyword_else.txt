identifier ('b')
whitespace (' ')
= ('=')
whitespace (' ')
number ('1')
whitespace ('\n')
if ('if')
whitespace (' ')
identifier ('b')
whitespace (' ')
> ('>')
whitespace (' ')
number ('2')
whitespace ('\n  ')
identifier ('b')
whitespace (' ')
= ('=')
whitespace (' ')
number ('10')
whitespace ('\n')
else ('else')
whitespace ('\n  ')
identifier ('b')
whitespace (' ')
= ('=')
whitespace (' ')
number ('20')
whitespace ('\n')
end ('end')